src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -351,3 +337,21 @@ function collapseIfPrivateSession() {
         $('#timeFramePanel, #instructionsRow, #responsesVisibleFromColumn').show();
     }
 }
+
+$(document).ready(function() {
+    var isEdit = typeof readyFeedbackEditPage === 'function';
+
+    if (typeof RichTextEditor !== 'undefined') {
+        /* eslint-disable camelcase */ // The property names are determined by external library (tinymce)
+        var richTextEditor = new RichTextEditor({
+            initParams: {
+                selector: '#instructions',
+                inline: true,
+                readonly: isEdit,
+                fixed_toolbar_container: '#richtext-toolbar-container'
+            }
+        });
+        richTextEditor.init();
+        /* eslint-enable camelcase */
+    }
+});
```

Comment:
Can you move this back up? `$(document).ready(...)` are all moved up as part of one of the refactorings done previously.